website/app/styles/doc-components/scroll-to-top.scss

```diff
@@ -0,0 +1,29 @@
+@use "../typography/mixins" as *;
+
+.doc-scroll-to-top {
+  position: fixed;
+  right: 40px;
+  bottom: 40px;
+  z-index: 99;
+  width: 44px;
+  height: 44px;
+  padding: 8px 8px 4px 8px;
+  background-color: rgba(52, 53, 54, 80%);
+  border: 2px solid transparent;
+  border-radius: 50%;
+  box-shadow: 0 8px 12px 0 rgba(37, 38, 45, 8%);
```

Comment:
@heatherlarsen quick question, to double check: in the Figma designs I see the opacity is applied to the whole button (so it includes the icon and the focus ring) while here is only applied to the background color; I imagine this was a choice done in the implementation phase, is it correct? in that case is it worth it to update the Figma file to match the new designs? (if not, @Dhaulagiri you have to apply the opacity to the whole button, not just the background)

Reply:
Ah, yeah, there were some tweaks to the styling during the implementation phase. It's probably a good idea to clean this up and have the opacity on the whole element instead of just the background color. Good catch!

Reply:
Fixed in https://github.com/hashicorp/design-system/pull/2080/commits/2c1abaaee2129d7c0ed60ded1bf67c6c3a0148e7